test/extended/cluster/cl.go

```diff
@@ -0,0 +1,147 @@
+package cluster
+
+import (
+	"fmt"
+	"path/filepath"
+	"strconv"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+
+	oapi "github.com/openshift/origin/pkg/api"
+	projectapi "github.com/openshift/origin/pkg/project/api"
+	exutil "github.com/openshift/origin/test/extended/util"
+	testutil "github.com/openshift/origin/test/util"
+	kapi "k8s.io/kubernetes/pkg/api"
+	kclientset "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
+)
+
+const deploymentRunTimeout = 5 * time.Minute
+
+var _ = g.Describe("clusterloader", func() {
+	defer g.GinkgoRecover()
+	var (
+		oc = exutil.NewCLI("cl", exutil.KubeConfigPath())
+		//deploymentFixture = exutil.FixturePath("testdata", "deployments", "test-deployment-test.yaml")
+	)
+
+	var c *kclientset.Clientset
+	g.BeforeEach(func() {
+		c = oc.AdminKubeClient()
+		ParseConfig(e2e.TestContext.Viper)
+	})
+
+	g.It("Should load the cluster", func() {
+
+		project := ConfigContext.ClusterLoader.Projects
```

Comment:
`project` is the bulk of the definition in the config file outlining how many different namespaces are to be created as well as the objects inside each of the namespaces. If you have some time Monday I would appreciate some further clarity on this.

In the kubernetes version the API would return namespace objects on creation and I would keep those for cleanup. 

L61 remnants `namespaces = append(namespaces, ns)`

I need to find a way to do that here.